app/src/main/java/com/monstarlab/core/extensions/FlowExtensions.kt

```diff
@@ -1,65 +1,16 @@
 package com.monstarlab.core.extensions
 
-import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.collect
-import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.flow
-import kotlinx.coroutines.launch
+import kotlin.time.Duration
 
-fun <T1, T2> CoroutineScope.combineFlows(
-    flow1: Flow<T1>,
-    flow2: Flow<T2>,
-    collectBlock: (suspend (T1, T2) -> Unit)
-) {
-    launch {
-        flow1.combine(flow2) { v1, v2 ->
-            collectBlock.invoke(v1, v2)
-        }.collect {
-            // Empty collect block to trigger ^
-        }
-    }
-}
-
-fun <T1, T2, T3> CoroutineScope.combineFlows(
-    flow1: Flow<T1>,
-    flow2: Flow<T2>,
-    flow3: Flow<T3>,
-    collectBlock: (suspend (T1, T2, T3) -> Unit)
-) {
-    launch {
-        combine(flow1, flow2, flow3) { v1, v2, v3 ->
-            collectBlock.invoke(v1, v2, v3)
-        }.collect {
-            // Empty collect block to trigger ^
-        }
-    }
-}
-
-fun <T1, T2, T3, T4> CoroutineScope.combineFlows(
-    flow1: Flow<T1>,
-    flow2: Flow<T2>,
-    flow3: Flow<T3>,
-    flow4: Flow<T4>,
-    collectBlock: (suspend (T1, T2, T3, T4) -> Unit)
-) {
-    launch {
-        combine(flow1, flow2, flow3, flow4) { v1, v2, v3, v4 ->
-            collectBlock.invoke(v1, v2, v3, v4)
-        }.collect {
-            // Empty collect block to trigger ^
-        }
-    }
-}
-
-fun <T> Flow<T>.throttleFirst(windowDuration: Long): Flow<T> = flow {
-    var lastEmissionTime = 0L
-    collect { upstream ->
-        val currentTime = System.currentTimeMillis()
-        val mayEmit = currentTime - lastEmissionTime > windowDuration
-        if (mayEmit) {
-            lastEmissionTime = currentTime
-            emit(upstream)
+fun tickerFlow(interval: Duration, initialDelay: Duration = interval): Flow<Unit> {
```

Comment:
I think `initialDelay` as 0 by default makes more sense than being the same than the interval 🤔

Reply:
I was on a fence with that, will update 👍